src/cli.js

```diff
@@ -313,6 +313,11 @@ async function setupCLI() {
             description: 'If true, will not write CHANGELOG.md updates for each package that has changed',
             type: 'boolean',
           })
+          .option('noCommit', {
+            default: false,
+            description: 'If true, will commit any changed files and leave them uncommitted',
```

Comment:
is this description accurate? Suggest changing to `If true, will modify all required files but leave them uncommitted after all operations as completed. This will also prevent a git push from occuring`